_src/static/sass/components/_buttons.scss

```diff
@@ -0,0 +1,371 @@
+// ------------------------------
+// edX Pattern Library: Components - Buttons
+
+// About: Contains base styling for demonstration and utility for buttons.
+// ----------------------------
+
+// #LAYOUT
+// #GLOBALS
+// #BASE (size)
+// #LARGE (size)
+// #SMALL (size)
+// #DEFAULT (style)
+// #DEFAULT-ALT (style)
+// #PRIMARY (style)
+// #SECONDARY (style)
+// #SECONDARY-ALT (style)
+// #LINK (style)
+// #ELEVATED (type)
+// #COMBO (type)
+// #ICONS (type)
+// #OVERLAY (type)
+// ----------------------------
+
+// ----------------------------
+// #LAYOUT
+// ----------------------------
+.example-set {
```

Comment:
Hmm, this shouldn't be in the component/element rendering. A better place is in the ``/src/pl/static/...`` Sass, since this is styling that is just to render these buttons in the preview/doc site. Please move it to there (if its not already there).

Oh, and if you're following bad form/habits from some previous work, mind removing any of the same from other component Sass?